addons/product/static/src/product_catalog/order_line/order_line.js

```diff
@@ -47,4 +47,8 @@ export class ProductCatalogOrderLine extends Component {
         const options = { digits, decimalPoint: ".", thousandsSep: "" };
         return parseFloat(formatFloat(this.props.quantity, options));
     }
+
+    get showPrice() {
+        return this.env.orderResModel !== "mrp.production" && this.env.orderResModel !== "repair.order";
```

Comment:
This doesn't look modular. Shouldn't this method return `true` in `product` and be overridden in `mrp` and/or `repair`? Like it is done for `disabledButtonTooltip` just above, overridden in `sale_stock`.

Reply:
Makes sense :+1: I updated it.